### ISSUE TITLE
Diagnose and fix Jacoco coverage in the infrastructure

### ISSUE THREAD

**Issue by xdevroey** [JUnitContest/junitcontest#1]
The Jacoco coverage does not work properly with some generated tests.

**Comment by xdevroey** [comment]
For instance, test cases generated by EvoSuite seems to fail for the coverage measurement with Jacoco (see [last year's results](https://github.com/PROSRESEARCHCENTER/junitcontest/blob/master/publications/SBSTcontest2019_detailed_results.pdf)).

**Comment by xdevroey** [comment]
Here is also a stack trace reported by Jose Campos last year:
```
.../home/evosuite/results_evosuite_10/ANTLR-10_2/metrics/log_detailed.txt
...
=== Run Jacoco for Coverage ===
Could not calculate coverage metrics!
java.io.IOException: Error while instrumenting @org/antlr/v4/unicode/UnicodeData.class.
	at org.jacoco.core.instr.Instrumenter.instrumentError(Instrumenter.java:170)
	at org.jacoco.core.instr.Instrumenter.instrument(Instrumenter.java:120)
	at org.jacoco.core.instr.Instrumenter.instrument(Instrumenter.java:145)
	at org.jacoco.core.instr.Instrumenter.instrument(Instrumenter.java:165)
	at org.jacoco.core.instr.Instrumenter.instrumentAll(Instrumenter.java:204)
	at org.jacoco.core.instr.Instrumenter.instrumentZip(Instrumenter.java:232)
	at org.jacoco.core.instr.Instrumenter.instrumentAll(Instrumenter.java:207)
	at sbst.benchmark.coverage.JaCoCoLauncher.runJaCoCo(JaCoCoLauncher.java:181)
	at sbst.benchmark.TestSuite.jacoco(TestSuite.java:454)
	at sbst.benchmark.TestSuite.<init>(TestSuite.java:97)
	at sbst.benchmark.TranscriptWriter.endClass(TranscriptWriter.java:157)
	at sbst.benchmark.RunTool.execute(RunTool.java:172)
	at sbst.benchmark.RunTool.run(RunTool.java:95)
	at sbst.benchmark.Main.main(Main.java:178)
Caused by: org.objectweb.asm.MethodTooLargeException: Method too large: org/antlr/v4/unicode/UnicodeData.addPropertyAliases ()V
	at org.objectweb.asm.MethodWriter.computeMethodInfoSize(MethodWriter.java:2089)
	at org.objectweb.asm.ClassWriter.toByteArray(ClassWriter.java:458)
	at org.jacoco.core.instr.Instrumenter.instrument(Instrumenter.java:101)
	at org.jacoco.core.instr.Instrumenter.instrument(Instrumenter.java:118)
	... 12 more
```

**Comment by xdevroey** [comment]
See branch https://github.com/JUnitContest/junitcontest/tree/fixing-coverage

**Comment by xdevroey** [comment]
Fixed in #9
